index.html

```diff
@@ -29,63 +29,211 @@
+                        href="https://stackoverflow.com/users/login?ssrc=anon_ask&returnurl=https%3a%2f%2fstackoverflow.com%2fquestions%2fask">
+                        Ask Question </a>
+                </div>
+                <div class="flex justify-between">
+                    <p class="font-18 font-weight-light"> 23,161,837 questions </p>
+                    <div class="flex align-center filter-question">
+                        <a class="btn btn-gray font-15 px-3 py-2" href="https://stackoverflow.com/questions?tab=Newest">
+                            Newest
+                        </a>
+                        <a class="btn btn-gray font-15 px-3 py-2" href="https://stackoverflow.com/questions?tab=Active">
+                            Active
+                        </a>
+                        <a class="btn btn-gray font-15 px-3 py-2"
+                            href="https://stackoverflow.com/questions?tab=Bounties">
+                            Bountied </a>
+                        <a class="btn btn-gray font-15 px-3 py-2"
+                            href="https://stackoverflow.com/questions?tab=Unanswered">
+                            Unanswered
+                        </a>
+                        <p class="btn btn-gray font-15 right-radius px-3 py-2 btn-icon gray"> More <svg
+                                xmlns="http://www.w3.org/2000/svg" viewBox="0 0 320 512">
+                                <!--! Font Awesome Pro 6.2.0 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license (Commercial License) Copyright 2022 Fonticons, Inc. -->
+                                <path
+                                    d="M137.4 374.6c12.5 12.5 32.8 12.5 45.3 0l128-128c9.2-9.2 11.9-22.9 6.9-34.9s-16.6-19.8-29.6-19.8L32 192c-12.9 0-24.6 7.8-29.6 19.8s-2.2 25.7 6.9 34.9l128 128z " />
+                            </svg> </p>
 
+                        <p class="btn btn-lightblue ml-4 font-15 px-3 py-2 btn-icon navy"> <svg
+                                xmlns="http://www.w3.org/2000/svg" viewBox="0 0 576 512">
+                                <!--! Font Awesome Pro 6.2.0 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license (Commercial License) Copyright 2022 Fonticons, Inc. -->
+                                <path
+                                    d="M151.6 469.6C145.5 476.2 137 480 128 480s-17.5-3.8-23.6-10.4l-88-96c-11.9-13-11.1-33.3 2-45.2s33.3-11.1 45.2 2L96 365.7V64c0-17.7 14.3-32 32-32s32 14.3 32 32V365.7l32.4-35.4c11.9-13 32.2-13.9 45.2-2s13.9 32.2 2 45.2l-88 96zM320 480c-17.7 0-32-14.3-32-32s14.3-32 32-32h32c17.7 0 32 14.3 32 32s-14.3 32-32 32H320zm0-128c-17.7 0-32-14.3-32-32s14.3-32 32-32h96c17.7 0 32 14.3 32 32s-14.3 32-32 32H320zm0-128c-17.7 0-32-14.3-32-32s14.3-32 32-32H480c17.7 0 32 14.3 32 32s-14.3 32-32 32H320zm0-128c-17.7 0-32-14.3-32-32s14.3-32 32-32H544c17.7 0 32 14.3 32 32s-14.3 32-32 32H320z" />
+                            </svg> Filter </p>
```

Comment:
These should not be `<p>` but buttons. You're meant to click on them, then something happens.